### ISSUE TITLE
Problems syncing via OSX 10.10 Contacts application

### ISSUE THREAD

**Issue by spidernik84** [Kozea/Radicale#269]
Hello, I'm happily using Radicale 0.10. I have a setup served by nginx and uwsgi with imap authentication. I used this nice guide http://www.robotamer.com/linux/radicale.html.

I can sync calendar entries and contact entries fine with this platforms/applications:
- Syncevolution on Jolla Sailfish (contacts + calendar)
- Sogo connector on OSX (contacts)
- iCal on OSX (calendar)
- Lightning on OSX (calendar)

So, I suppose the setup is right. The issue is with OSX Contacts. When attempting to setup the carddav resource I receive mixed errors, most notably
"Unable to verify account name or password"
"A secure connection could not be established"

Both make no sense! I'm using a valid startssl certificate and the pass/user are ok.

I'm providing the enormous debug log. I see some "anonymous access" attempts but I can't see why since I'm providing user and pass.

Thanks!

```
2015-03-15 11:59:27,629 - DEBUG: Authentication type is IMAP
2015-03-15 11:59:49,554 - INFO: PROPFIND request at /radicale/info@mydomain.net/Address.vcf/ received
2015-03-15 11:59:49,555 - DEBUG: Request headers:
{'CONTENT_LENGTH': '181',
 'CONTENT_TYPE': 'text/xml',
 'DOCUMENT_ROOT': '/var/www/mydomain.net',
 'HTTPS': 'on',
 'HTTP_ACCEPT': '*/*',
 'HTTP_ACCEPT_ENCODING': 'gzip, deflate',
 'HTTP_ACCEPT_LANGUAGE': 'en-us',
 'HTTP_BRIEF': 't',
 'HTTP_CONTENT_LENGTH': '181',
 'HTTP_CONTENT_TYPE': 'text/xml',
 'HTTP_DEPTH': '0',
 'HTTP_HOST': 'mydomain.net',
 'HTTP_PREFER': 'return=minimal',
 'HTTP_USER_AGENT': 'Mac OS X/10.10.2 (14C109) AddressBook/1563',
 'PATH_INFO': '/radicale/info@mydomain.net/Address.vcf/',
 'QUERY_STRING': '',
 'REMOTE_ADDR': '213.113.32.202',
 'REMOTE_PORT': '55087',
 'REQUEST_METHOD': 'PROPFIND',
 'REQUEST_URI': '/radicale/info%40mydomain.net/Address.vcf/',
 'SERVER_NAME': 'mydomain.net',
 'SERVER_PORT': '443',
 'SERVER_PROTOCOL': 'HTTP/1.1',
 'uwsgi.node': 'mail',
 'uwsgi.version': '1.0.3-debian',
 'wsgi.errors': <open file 'wsgi_errors', mode 'w' at 0x22f1810>,
 'wsgi.file_wrapper': <built-in function uwsgi_sendfile>,
 'wsgi.input': <uwsgi._Input object at 0x238f510>,
 'wsgi.multiprocess': True,
 'wsgi.multithread': False,
 'wsgi.run_once': False,
 'wsgi.url_scheme': 'https',
 'wsgi.version': (1, 0)}
2015-03-15 11:59:49,556 - DEBUG: Sanitized path: /radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,557 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:49,558 - DEBUG: Test if ':radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:49,558 - DEBUG: Anonymous has NO read access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,559 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:49,559 - DEBUG: Test if ':radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:49,559 - DEBUG: Anonymous has NO write access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,560 - DEBUG: Request content:
<?xml version="1.0" encoding="UTF-8"?>
<A:propfind xmlns:A="DAV:">
  <A:prop>
    <A:current-user-principal/>
    <A:principal-URL/>
    <A:resourcetype/>
  </A:prop>
</A:propfind>

2015-03-15 11:59:49,560 - INFO: Anonymous user refused
2015-03-15 11:59:49,560 - DEBUG: Answer status: 401 Unauthorized
2015-03-15 11:59:49,594 - INFO: PROPFIND request at /radicale/info@mydomain.net/Address.vcf/ received
2015-03-15 11:59:49,595 - DEBUG: Request headers:
{'CONTENT_LENGTH': '181',
 'CONTENT_TYPE': 'text/xml',
 'DOCUMENT_ROOT': '/var/www/mydomain.net',
 'HTTPS': 'on',
 'HTTP_ACCEPT': '*/*',
 'HTTP_ACCEPT_ENCODING': 'gzip, deflate',
 'HTTP_ACCEPT_LANGUAGE': 'en-us',
 'HTTP_BRIEF': 't',
 'HTTP_CONTENT_LENGTH': '181',
 'HTTP_CONTENT_TYPE': 'text/xml',
 'HTTP_DEPTH': '0',
 'HTTP_HOST': 'mydomain.net',
 'HTTP_PREFER': 'return=minimal',
 'HTTP_USER_AGENT': 'Mac OS X/10.10.2 (14C109) AddressBook/1563',
 'PATH_INFO': '/radicale/info@mydomain.net/Address.vcf/',
 'QUERY_STRING': '',
 'REMOTE_ADDR': '213.113.32.202',
 'REMOTE_PORT': '55087',
 'REQUEST_METHOD': 'PROPFIND',
 'REQUEST_URI': '/radicale/info%40mydomain.net/Address.vcf/',
 'SERVER_NAME': 'mydomain.net',
 'SERVER_PORT': '443',
 'SERVER_PROTOCOL': 'HTTP/1.1',
 'uwsgi.node': 'mail',
 'uwsgi.version': '1.0.3-debian',
 'wsgi.errors': <open file 'wsgi_errors', mode 'w' at 0x22f1810>,
 'wsgi.file_wrapper': <built-in function uwsgi_sendfile>,
 'wsgi.input': <uwsgi._Input object at 0x238f510>,
 'wsgi.multiprocess': True,
 'wsgi.multithread': False,
 'wsgi.run_once': False,
 'wsgi.url_scheme': 'https',
 'wsgi.version': (1, 0)}
2015-03-15 11:59:49,596 - DEBUG: Sanitized path: /radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,598 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:49,598 - DEBUG: Test if ':radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:49,599 - DEBUG: Anonymous has NO read access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,599 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:49,599 - DEBUG: Test if ':radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:49,599 - DEBUG: Anonymous has NO write access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,600 - DEBUG: Request content:
<?xml version="1.0" encoding="UTF-8"?>
<A:propfind xmlns:A="DAV:">
  <A:prop>
    <A:current-user-principal/>
    <A:principal-URL/>
    <A:resourcetype/>
  </A:prop>
</A:propfind>

2015-03-15 11:59:49,601 - INFO: Anonymous user refused
2015-03-15 11:59:49,601 - DEBUG: Answer status: 401 Unauthorized
2015-03-15 11:59:49,635 - INFO: PROPFIND request at /radicale/info@mydomain.net/Address.vcf/ received
2015-03-15 11:59:49,636 - DEBUG: Request headers:
{'CONTENT_LENGTH': '181',
 'CONTENT_TYPE': 'text/xml',
 'DOCUMENT_ROOT': '/var/www/mydomain.net',
 'HTTPS': 'on',
 'HTTP_ACCEPT': '*/*',
 'HTTP_ACCEPT_ENCODING': 'gzip, deflate',
 'HTTP_ACCEPT_LANGUAGE': 'en-us',
 'HTTP_AUTHORIZATION': 'Basic aW5mb0BuaWNvbGF2b2xwaW5pLm5ldDpDNG4zUDR6ejAhMTk4NA==',
 'HTTP_BRIEF': 't',
 'HTTP_CONTENT_LENGTH': '181',
 'HTTP_CONTENT_TYPE': 'text/xml',
 'HTTP_DEPTH': '0',
 'HTTP_HOST': 'mydomain.net',
 'HTTP_PREFER': 'return=minimal',
 'HTTP_USER_AGENT': 'Mac OS X/10.10.2 (14C109) AddressBook/1563',
 'PATH_INFO': '/radicale/info@mydomain.net/Address.vcf/',
 'QUERY_STRING': '',
 'REMOTE_ADDR': '213.113.32.202',
 'REMOTE_PORT': '55087',
 'REQUEST_METHOD': 'PROPFIND',
 'REQUEST_URI': '/radicale/info%40mydomain.net/Address.vcf/',
 'SERVER_NAME': 'mydomain.net',
 'SERVER_PORT': '443',
 'SERVER_PROTOCOL': 'HTTP/1.1',
 'uwsgi.node': 'mail',
 'uwsgi.version': '1.0.3-debian',
 'wsgi.errors': <open file 'wsgi_errors', mode 'w' at 0x22f1810>,
 'wsgi.file_wrapper': <built-in function uwsgi_sendfile>,
 'wsgi.input': <uwsgi._Input object at 0x238f510>,
 'wsgi.multiprocess': True,
 'wsgi.multithread': False,
 'wsgi.run_once': False,
 'wsgi.url_scheme': 'https',
 'wsgi.version': (1, 0)}
2015-03-15 11:59:49,637 - DEBUG: Sanitized path: /radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,637 - DEBUG: Connecting to IMAP server localhost:143.
2015-03-15 11:59:49,650 - WARNING: IMAP server is local. Will allow transmitting unencrypted credentials.
2015-03-15 11:59:49,663 - DEBUG: Authenticated IMAP user info@mydomain.net via localhost.
2015-03-15 11:59:49,663 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:49,663 - DEBUG: Test if 'info@mydomain.net:radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:49,663 - DEBUG: Section 'rw' matches
2015-03-15 11:59:49,663 - DEBUG: info@mydomain.net has read access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,664 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:49,664 - DEBUG: Test if 'info@mydomain.net:radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:49,664 - DEBUG: Section 'rw' matches
2015-03-15 11:59:49,664 - DEBUG: info@mydomain.net has write access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,664 - DEBUG: Request content:
<?xml version="1.0" encoding="UTF-8"?>
<A:propfind xmlns:A="DAV:">
  <A:prop>
    <A:current-user-principal/>
    <A:principal-URL/>
    <A:resourcetype/>
  </A:prop>
</A:propfind>

2015-03-15 11:59:49,667 - DEBUG: Response content:
<?xml version="1.0"?>
<multistatus xmlns="DAV:" xmlns:CR="urn:ietf:params:xml:ns:carddav">
  <response>
    <href>/radicale/info@mydomain.net/Address.vcf/</href>
    <propstat>
      <prop>
        <current-user-principal>
          <href>/info@mydomain.net/</href>
        </current-user-principal>
        <principal-URL>
          <href>/radicale/info@mydomain.net/Address.vcf/</href>
        </principal-URL>
        <resourcetype>
          <CR:addressbook />
          <collection />
        </resourcetype>
      </prop>
      <status>HTTP/1.1 200 OK</status>
    </propstat>
  </response>
</multistatus>

2015-03-15 11:59:49,668 - DEBUG: Answer status: 207 Unknown
2015-03-15 11:59:49,676 - INFO: PROPFIND request at /radicale/info@mydomain.net/Address.vcf/ received
2015-03-15 11:59:49,678 - DEBUG: Request headers:
{'CONTENT_LENGTH': '181',
 'CONTENT_TYPE': 'text/xml',
 'DOCUMENT_ROOT': '/var/www/mydomain.net',
 'HTTPS': 'on',
 'HTTP_ACCEPT': '*/*',
 'HTTP_ACCEPT_ENCODING': 'gzip, deflate',
 'HTTP_ACCEPT_LANGUAGE': 'en-us',
 'HTTP_AUTHORIZATION': 'Basic aW5mb0BuaWNvbGF2b2xwaW5pLm5ldDpDNG4zUDR6ejAhMTk4NA==',
 'HTTP_BRIEF': 't',
 'HTTP_CONTENT_LENGTH': '181',
 'HTTP_CONTENT_TYPE': 'text/xml',
 'HTTP_DEPTH': '0',
 'HTTP_HOST': 'mydomain.net',
 'HTTP_PREFER': 'return=minimal',
 'HTTP_USER_AGENT': 'Mac OS X/10.10.2 (14C109) AddressBook/1563',
 'PATH_INFO': '/radicale/info@mydomain.net/Address.vcf/',
 'QUERY_STRING': '',
 'REMOTE_ADDR': '213.113.32.202',
 'REMOTE_PORT': '55087',
 'REQUEST_METHOD': 'PROPFIND',
 'REQUEST_URI': '/radicale/info%40mydomain.net/Address.vcf/',
 'SERVER_NAME': 'mydomain.net',
 'SERVER_PORT': '443',
 'SERVER_PROTOCOL': 'HTTP/1.1',
 'uwsgi.node': 'mail',
 'uwsgi.version': '1.0.3-debian',
 'wsgi.errors': <open file 'wsgi_errors', mode 'w' at 0x22f1810>,
 'wsgi.file_wrapper': <built-in function uwsgi_sendfile>,
 'wsgi.input': <uwsgi._Input object at 0x238f510>,
 'wsgi.multiprocess': True,
 'wsgi.multithread': False,
 'wsgi.run_once': False,
 'wsgi.url_scheme': 'https',
 'wsgi.version': (1, 0)}
2015-03-15 11:59:49,679 - DEBUG: Sanitized path: /radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,680 - DEBUG: Connecting to IMAP server localhost:143.
2015-03-15 11:59:49,689 - WARNING: IMAP server is local. Will allow transmitting unencrypted credentials.
2015-03-15 11:59:49,696 - DEBUG: Authenticated IMAP user info@mydomain.net via localhost.
2015-03-15 11:59:49,696 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:49,696 - DEBUG: Test if 'info@mydomain.net:radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:49,697 - DEBUG: Section 'rw' matches
2015-03-15 11:59:49,697 - DEBUG: info@mydomain.net has read access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,697 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:49,697 - DEBUG: Test if 'info@mydomain.net:radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:49,698 - DEBUG: Section 'rw' matches
2015-03-15 11:59:49,698 - DEBUG: info@mydomain.net has write access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:49,699 - DEBUG: Request content:
<?xml version="1.0" encoding="UTF-8"?>
<A:propfind xmlns:A="DAV:">
  <A:prop>
    <A:current-user-principal/>
    <A:principal-URL/>
    <A:resourcetype/>
  </A:prop>
</A:propfind>

2015-03-15 11:59:49,704 - DEBUG: Response content:
<?xml version="1.0"?>
<multistatus xmlns="DAV:" xmlns:CR="urn:ietf:params:xml:ns:carddav">
  <response>
    <href>/radicale/info@mydomain.net/Address.vcf/</href>
    <propstat>
      <prop>
        <current-user-principal>
          <href>/info@mydomain.net/</href>
        </current-user-principal>
        <principal-URL>
          <href>/radicale/info@mydomain.net/Address.vcf/</href>
        </principal-URL>
        <resourcetype>
          <CR:addressbook />
          <collection />
        </resourcetype>
      </prop>
      <status>HTTP/1.1 200 OK</status>
    </propstat>
  </response>
</multistatus>

2015-03-15 11:59:49,705 - DEBUG: Answer status: 207 Unknown
2015-03-15 11:59:54,859 - INFO: PROPFIND request at /radicale/info@mydomain.net/Address.vcf/ received
2015-03-15 11:59:54,860 - DEBUG: Request headers:
{'CONTENT_LENGTH': '181',
 'CONTENT_TYPE': 'text/xml',
 'DOCUMENT_ROOT': '/var/www/mydomain.net',
 'HTTPS': 'on',
 'HTTP_ACCEPT': '*/*',
 'HTTP_ACCEPT_ENCODING': 'gzip, deflate',
 'HTTP_ACCEPT_LANGUAGE': 'en-us',
 'HTTP_AUTHORIZATION': 'Basic aW5mb0BuaWNvbGF2b2xwaW5pLm5ldDpDNG4zUDR6ejAhMTk4NA==',
 'HTTP_BRIEF': 't',
 'HTTP_CONTENT_LENGTH': '181',
 'HTTP_CONTENT_TYPE': 'text/xml',
 'HTTP_DEPTH': '0',
 'HTTP_HOST': 'mydomain.net',
 'HTTP_PREFER': 'return=minimal',
 'HTTP_USER_AGENT': 'Mac OS X/10.10.2 (14C109) AddressBook/1563',
 'PATH_INFO': '/radicale/info@mydomain.net/Address.vcf/',
 'QUERY_STRING': '',
 'REMOTE_ADDR': '213.113.32.202',
 'REMOTE_PORT': '55087',
 'REQUEST_METHOD': 'PROPFIND',
 'REQUEST_URI': '/radicale/info%40mydomain.net/Address.vcf/',
 'SERVER_NAME': 'mydomain.net',
 'SERVER_PORT': '443',
 'SERVER_PROTOCOL': 'HTTP/1.1',
 'uwsgi.node': 'mail',
 'uwsgi.version': '1.0.3-debian',
 'wsgi.errors': <open file 'wsgi_errors', mode 'w' at 0x22f1810>,
 'wsgi.file_wrapper': <built-in function uwsgi_sendfile>,
 'wsgi.input': <uwsgi._Input object at 0x238f510>,
 'wsgi.multiprocess': True,
 'wsgi.multithread': False,
 'wsgi.run_once': False,
 'wsgi.url_scheme': 'https',
 'wsgi.version': (1, 0)}
2015-03-15 11:59:54,860 - DEBUG: Sanitized path: /radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:54,860 - DEBUG: Connecting to IMAP server localhost:143.
2015-03-15 11:59:54,870 - DEBUG: Authenticated IMAP user info@mydomain.net via localhost.
2015-03-15 11:59:54,870 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:54,870 - DEBUG: Test if 'info@mydomain.net:radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:54,870 - DEBUG: Section 'rw' matches
2015-03-15 11:59:54,871 - DEBUG: info@mydomain.net has read access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:54,871 - DEBUG: Rights type 'authenticated'
2015-03-15 11:59:54,871 - DEBUG: Test if 'info@mydomain.net:radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 11:59:54,871 - DEBUG: Section 'rw' matches
2015-03-15 11:59:54,871 - DEBUG: info@mydomain.net has write access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 11:59:54,872 - DEBUG: Request content:
<?xml version="1.0" encoding="UTF-8"?>
<A:propfind xmlns:A="DAV:">
  <A:prop>
    <A:current-user-principal/>
    <A:principal-URL/>
    <A:resourcetype/>
  </A:prop>
</A:propfind>

2015-03-15 11:59:54,873 - DEBUG: Response content:
<?xml version="1.0"?>
<multistatus xmlns="DAV:" xmlns:CR="urn:ietf:params:xml:ns:carddav">
  <response>
    <href>/radicale/info@mydomain.net/Address.vcf/</href>
    <propstat>
      <prop>
        <current-user-principal>
          <href>/info@mydomain.net/</href>
        </current-user-principal>
        <principal-URL>
          <href>/radicale/info@mydomain.net/Address.vcf/</href>
        </principal-URL>
        <resourcetype>
          <CR:addressbook />
          <collection />
        </resourcetype>
      </prop>
      <status>HTTP/1.1 200 OK</status>
    </propstat>
  </response>
</multistatus>

2015-03-15 11:59:54,873 - DEBUG: Answer status: 207 Unknown
2015-03-15 12:00:09,156 - INFO: PROPFIND request at /radicale/info@mydomain.net/Address.vcf/ received
2015-03-15 12:00:09,157 - DEBUG: Request headers:
{'CONTENT_LENGTH': '181',
 'CONTENT_TYPE': 'text/xml',
 'DOCUMENT_ROOT': '/var/www/mydomain.net',
 'HTTPS': 'on',
 'HTTP_ACCEPT': '*/*',
 'HTTP_ACCEPT_ENCODING': 'gzip, deflate',
 'HTTP_ACCEPT_LANGUAGE': 'en-us',
 'HTTP_AUTHORIZATION': 'Basic aW5mb0BuaWNvbGF2b2xwaW5pLm5ldDpDNG4zUDR6ejAhMTk4NA==',
 'HTTP_BRIEF': 't',
 'HTTP_CONTENT_LENGTH': '181',
 'HTTP_CONTENT_TYPE': 'text/xml',
 'HTTP_DEPTH': '0',
 'HTTP_HOST': 'mydomain.net',
 'HTTP_PREFER': 'return=minimal',
 'HTTP_USER_AGENT': 'Mac OS X/10.10.2 (14C109) AddressBook/1563',
 'PATH_INFO': '/radicale/info@mydomain.net/Address.vcf/',
 'QUERY_STRING': '',
 'REMOTE_ADDR': '213.113.32.202',
 'REMOTE_PORT': '55087',
 'REQUEST_METHOD': 'PROPFIND',
 'REQUEST_URI': '/radicale/info%40mydomain.net/Address.vcf/',
 'SERVER_NAME': 'mydomain.net',
 'SERVER_PORT': '443',
 'SERVER_PROTOCOL': 'HTTP/1.1',
 'uwsgi.node': 'mail',
 'uwsgi.version': '1.0.3-debian',
 'wsgi.errors': <open file 'wsgi_errors', mode 'w' at 0x22f1810>,
 'wsgi.file_wrapper': <built-in function uwsgi_sendfile>,
 'wsgi.input': <uwsgi._Input object at 0x238f510>,
 'wsgi.multiprocess': True,
 'wsgi.multithread': False,
 'wsgi.run_once': False,
 'wsgi.url_scheme': 'https',
 'wsgi.version': (1, 0)}
2015-03-15 12:00:09,158 - DEBUG: Sanitized path: /radicale/info@mydomain.net/Address.vcf/
2015-03-15 12:00:09,158 - DEBUG: Connecting to IMAP server localhost:143.
2015-03-15 12:00:09,171 - DEBUG: Authenticated IMAP user info@mydomain.net via localhost.
2015-03-15 12:00:09,171 - DEBUG: Rights type 'authenticated'
2015-03-15 12:00:09,171 - DEBUG: Test if 'info@mydomain.net:radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 12:00:09,171 - DEBUG: Section 'rw' matches
2015-03-15 12:00:09,171 - DEBUG: info@mydomain.net has read access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 12:00:09,171 - DEBUG: Rights type 'authenticated'
2015-03-15 12:00:09,172 - DEBUG: Test if 'info@mydomain.net:radicale/info@mydomain.net/Address.vcf' matches against '.+:.*' from section 'rw'
2015-03-15 12:00:09,172 - DEBUG: Section 'rw' matches
2015-03-15 12:00:09,172 - DEBUG: info@mydomain.net has write access to collection radicale/info@mydomain.net/Address.vcf/
2015-03-15 12:00:09,172 - DEBUG: Request content:
<?xml version="1.0" encoding="UTF-8"?>
<A:propfind xmlns:A="DAV:">
  <A:prop>
    <A:current-user-principal/>
    <A:principal-URL/>
    <A:resourcetype/>
  </A:prop>
</A:propfind>

2015-03-15 12:00:09,174 - DEBUG: Response content:
<?xml version="1.0"?>
<multistatus xmlns="DAV:" xmlns:CR="urn:ietf:params:xml:ns:carddav">
  <response>
    <href>/radicale/info@mydomain.net/Address.vcf/</href>
    <propstat>
      <prop>
        <current-user-principal>
          <href>/info@mydomain.net/</href>
        </current-user-principal>
        <principal-URL>
          <href>/radicale/info@mydomain.net/Address.vcf/</href>
        </principal-URL>
        <resourcetype>
          <CR:addressbook />
          <collection />
        </resourcetype>
      </prop>
      <status>HTTP/1.1 200 OK</status>
    </propstat>
  </response>
</multistatus>

2015-03-15 12:00:09,175 - DEBUG: Answer status: 207 Unknown

```

**Comment by tonioo** [comment]
Hi, I encounter the same issue with calendars (using thunderbird). I'm never prompted to authenticate so Radicale is unable to apply rights (see Anonymous in the log). I wonder if it is possible to launch the authentication prompt on the Python application side instead of the HTTP server one.

Any help about this ?

**Comment by tonioo** [comment]
Problem fixed for me. Was due to configuration mistake...

**Comment by liZe** [comment]
Did you find a solution for your problem?

**Comment by stephensprinkle** [comment]
I’d run into the exact same issue on my end

After observing the logs on Radicale’s end, it became clear this is more an issue w/ macOS Contacts rather than Radicale.

Steps to resolve (sorry, short of screenshots, this will just a step-by-step):

Open ‘Contacts’ App
Contacts > Preferences
Select the ‘Accounts’ Tab
Select ‘+’ to add a new account
Select 'Other Contacts Account…’
Make sure `CardDAV` is selected

This is where is gets tricky
Account Type: Advanced
Username: <username>
Password: <password>
Server Address: https://<hostname>
Server Path: /path/to/contacts.vcf/
Port: 5232
Use SSL Checked

This will fail reasoning authentication (it may also ask you to verify cert)

Now, select Account Type: Manual
UserName: <username>
Password: <password>
Server Address: <hostname>:5232

Try again to sign in and all should be well

When I observed the logs I found that the client had preserved the correct path and now contact sync/edit could be conducted as usual.

Hope this helps someone else who encountered the same issue!
